includes/Modules/Analytics.php

```diff
@@ -122,10 +122,21 @@ function() {
 		);
 
 		$print_amp_gtag = function() {
+			// This hook is only available in AMP plugin version 1.3+, so if it
+			// has already completed, do nothing.
+			if (
+				! doing_action( 'amp_print_analytics' ) &&
+				did_action( 'amp_print_analytics' )
```

Comment:
I think this is easier to read in one line, it's not too long.

Reply:
It wraps quite a lot on one line. But I'll go back to one line given there aren't any linter errors and you have a preference 😄